src/main/java/seedu/address/ui/HelpAddWindow.java

```diff
@@ -13,13 +13,19 @@
 /**
  * Controller for a help page.
  */
-public class HelpPoochSupplierWindow extends UiPart<Stage> {
+public class HelpAddWindow extends UiPart<Stage> {
 
     public static final String USERGUIDE_URL = "https://ay2324s2-cs2103t-w10-2.github.io/tp/UserGuide.html";
-    public static final String HELP_MESSAGE = "Adds a pooch supplier to pooch planner"
+    public static final String HELP_MESSAGE = "Adds a maintainer/supplier/staff/other person to pooch planner"
             + "\n" + "" + "\n"
+            + "Format:" + "\n"
+            + "/pooch-maintainer ; name : [name] ; phone : [phone] ; address : [address] ;"
+            + " email : [email] ; skill : [skill] ; commission : [commission/hr]" + "\n"
             + "/pooch-supplier ; name : [name] ; phone : [phone] ; address : [address] ;"
-            + " email : [email] ; product : [product] ; price : [price/(quantity)]"
+            + " email : [email] ; product : [product] ; price : [price/(quantity)]" + "\n"
+            + "/pooch-staff ; name : [name] ; phone : [phone] ; address : [address] ;"
+            + " email : [email] ; salary : [salary/hr]  ; employment : [part/full]" + "\n"
+            + "/pooch-add ; name : [name] ; phone : [phone] ; address : [address] ; email : [email]" + "\n"
```

Comment:
I think the basic one i.e. /pooch-add should be at the top since its the most general and the rest are specifics

Reply:
yes that makes sense!!